src/rustdoc-json-types/lib.rs

```diff
@@ -346,11 +346,8 @@ pub struct GenericParamDef {
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 #[serde(rename_all = "snake_case")]
 pub enum GenericParamDefKind {
-    Lifetime {
-        outlives: Vec<String>,
-    },
+    Lifetime,
```

Comment:
Why this change? You're removing a lot of information.

Reply:
Nevermind, just went through the rustdoc tests again and it seems to be working as expected.